config/config.go

```diff
@@ -193,6 +193,7 @@ type GlobalConfig struct {
 	SlackURL        string `yaml:"slack_url"`
 	PagerdutyURL    string `yaml:"pagerduty_url"`
 	OpsGenieAPIHost string `yaml:"opsgenie_api_host"`
+	ExternalURL     string `yaml:"external_url"`
```

Comment:
Everything that's trivial to change at runtime should not be a flag if there is a config file anyway.
I would rather change it in Prometheus.